### ISSUE TITLE
Rotating only one of the connected screens

### ISSUE THREAD

**Issue by musaoruc** [Bronzdragon/screen-rotator#2]
There should be an option to rotate only one of the connected monitors. Thank you for this extension.

**Comment by Bronzdragon** [comment]
Hey Musaoruc! Sorry for replying so late. I've been working a new job, so I've been busy.

You're right, I'd love to support that. Adding an option shouldn't be too hard. However, one of the issues I ran into is that all the monitors need to be touching, and not overlapping when I save the config. When you rotate a single monitor, you'd have to move it (or the other monitors) to make all the edges touch. And if you have a grid of monitors, fitting the newly rotated monitor in is kinda complicated.

You can probably think of the 'obvious' way a monitor should fit in with the others for any given configuration, but choosing the correct solution for every configuration is hard. On top of that, I'm not very confident with programming using the Gnome APIs. I barely managed to pull this together, if I'm honest.

To summarize: There isn't an obvious way a monitor fits in a multi-monitor setup, and so it's kind of a lot of effort to support.

**Comment by musaoruc** [comment]
That is okay thank you for your efforts anyway.
